src/views/VallisFish.vue

```diff
@@ -2,171 +2,141 @@
   <b-container fluid>
     <b-col md="12">
       <router-link to="/vallis/map">
-        <b-button variant="info" class="btn-block mb-3">Orb Vallis Map</b-button>
+        <b-btn variant="info" class="btn-block mb-3">Orb Vallis Map</b-btn>
       </router-link>
 
       <div class="row">
-        <b-table striped hover :items="items" class="fish-info b-table"></b-table>
+        <table class="table fish-info striped hover">
+          <thead>
+            <tr>
+              <th title="The name of the fish">Fish Name</th>
+              <th title="Number of scrap you receive is dependent on fish model">
+                Model
+              </th>
+              <th title="Number of scrap you will receive when dismantling">
```

Comment:
For better English, we could say "Amount of scrap" instead for this column and the previous one.